sdk/nodejs/iam/documents.ts

```diff
@@ -14,9 +14,11 @@
 
 import {Input} from "@pulumi/pulumi";
 
-// Implementation note: The definitions of `PolicyDocument` and its associated types below are based
-// on the grammar for IAM Policy Documents defined at
-// https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_grammar.html.
+/**
+ * Implementation note: The definitions of `PolicyDocument` and its associated types below are based
+ * on the grammar for IAM Policy Documents defined at
+ * https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_grammar.html.
+ */
```

Comment:
Nit: This should remain a regular comment since it isn't associated with a particular API, just a note about the implementation.